### PR TITLE
Feature/admin abilities

### PR DESCRIPTION
**Pull request by TylerRockwell**
### Why?
https://github.com/smashingboxes/apprenticeship/tree/master/Curriculum/Backend/bookstore
Because Admins need to create more Admins so they don't have to do everything themselves
Because `/admin/admins/new/` is an odd route

### What Changed?
Added `new` and `create` actions on an `AdminsController`
Changed `admin` namespace to `admin_api`

### CODE REVIEW

File: app/controllers/admin_api/admins_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      redirect_to admin_api_dashboard_path, alert: "Admin could not be saved"
+    end
+  end
+
+  private
+
+  def admin_params
+    params.require(:admin).permit(:email, :password, :password_confirmation)
+  end
+end
```
Comment (jahammo2) [inline app/controllers/admin_api/admins_controller.rb]: Controller looks good

File: app/controllers/admin_api/admins_controller.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class AdminApi::AdminsController < ApplicationController
```
Comment (jahammo2) [inline app/controllers/admin_api/admins_controller.rb]: I also like the use of namespacing here

**Comment by jahammo2** [comment]
Looks good Tyler. :shipit:

**Comment by TylerRockwell** [comment]
Thanks, @jahammo2

**Pull request merged**